conda/recipes/nvstrings/meta.yaml

```diff
@@ -1,7 +1,5 @@
 # Copyright (c) 2018, NVIDIA CORPORATION.
 
-# Usage:
-#   conda build -c defaults -c conda-forge .
 {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
 {% set git_revision_count=environ.get('GIT_DESCRIBE_NUMBER', 0) %}
 {% set cuda_version='.'.join(environ.get('CUDA_VERSION', 'unknown').split('.')[:2]) %}
```

Comment:
We should turn this into a split package at some point, but unsure if we want to tackle that in this PR or a future PR.

Reply:
Future PR